src/table/components/body/TableBodyWrapper.tsx

```diff
@@ -40,9 +40,12 @@ function TableBodyWrapper({
       ),
     [columnsStylingIDsJSON, isSelectionsEnabled]
   );
-  const bodyCellStyle = useMemo(() => getBodyCellStyle(layout, theme), [layout, theme]);
+  const { hoverColors, lastRowBottomBorder, ...cellStyle } = useMemo(
```

Comment:
Are you using spread (...) on `cellStyle` so that `hoverColors` or `lastRowBottomBorder` will be overwritten if they are `undefined`?

Reply:
I want do split them up into three different variables, so that we can use cellStyle for the cell, then the others are applied in other places

Reply:
👍